### DISCUSSION TITLE
Using PDF++ with external files

### DISCUSSION THREAD

**Discussion by antomonte** [RyotaUshio/obsidian-pdf-plus#60] [Q&A]
Hi,

your plugin is absolutely amazing!

Due to company privacy concerns, I try to avoid importing important PDFs into my vault.

Would it be possible to use the plugin with PDF files outside our vaults? (just located at our hard drives or company's cloud)

Best regards

**Comment by RyotaUshio** [comment]
Thank you for the kind words!

PDF++ can only handle PDF files that you can open in Obsidian. 
To open a PDF file outside the vault folder, you can use a symbolic link (I do this in my personal vault).

FYI, there are a few things to note when you use symbolic links in Obsidian: https://help.obsidian.md/Files+and+folders/Symbolic+links+and+junctions

**Reply by antomonte** [reply]
Many thanks for the suggestion, but we don't have admin privileges, so cannot use that technique.

**Reply by RyotaUshio** [reply]
On Linux or macOS, you can create symbolic links without `sudo`.
I'm not familiar with Windows but it seems that there is a workaround: https://stackoverflow.com/questions/58038683/allow-mklink-for-a-non-admin-user
Might be your interest.

**Reply by antomonte** [reply]
It seems that it is not allowed in my corporate environment (I've tried, unsuccessfully)
Anyways, your plugin is extremely useful. Keep it going!

**Reply by RyotaUshio** [reply]
@antomonte Hi, I guess I found a trick to open external PDFs inside Obsidian. Of course, we have to download the binary content of the PDF into our computer's memory but we don't need to save it to the hard disk

**Reply by antomonte** [reply]
Yeah? that would be really GREAT!!!
I'm looking forward being able to use it!

Actually, for me it would be enough just with pointing to any folder in my hard disk out of the vault, but if you can achieve to point everywhere.... yours will become one of the most important pieces for Obsidian users (if it isn't already!).

**Comment by RyotaUshio** [comment]
For those who will arrive at this thread in the future: now PDF++ can open external PDF files as well. Learn more at https://ryotaushio.github.io/obsidian-pdf-plus/external-pdf-files.html
